lib/dialects/postgres/query/pg-querycompiler.js

```diff
@@ -218,6 +218,18 @@ class QueryCompiler_PG extends QueryCompiler {
   distinctOn(value) {
     return 'distinct on (' + this.formatter.columnize(value) + ') ';
   }
+
+  whereLike(statement) {
```

Comment:
Do we still need client-specific implementations of this functions if we use standard knex escaping mechanism? I assume there are no other differences?